decoders/connector/lansitec/container-tracker/v1.0.0/payload.js

```diff
@@ -0,0 +1,396 @@
+// ContainerTracker
```

Comment:
New decoders that are added must be made in TypeScript and must have a unit test.